### PR TITLE
KPV-2609 :: Hided into mobile screens cCallToAction block view

### PR DESCRIPTION
**Pull request by pverdu-kuorum**

### CODE REVIEW

**Review by iduetxe — CHANGES_REQUESTED** [review]
Remove de render. It is not necessary. The button is printed at the end of the page

**Comment by pverdu-kuorum** [comment]
Removed.

File: web-app/js/campaigns/contest.js [hunk 1]
```diff
@@ -69,7 +69,7 @@ $(function () {
     $(".call-to-action").on("click", "a.btn.ADDING_APPLICATIONS", contestApplicationHelper.bindActionClickAddDistrictProposal);
 
     $(".actions.call-to-action-mobile.go-to-action button").on("click", function(e){
-        $(".actions.clearfix a")[0].click()
+        $(".actions.clearfix a")[1].click()
```
Comment (iduetxe) [inline web-app/js/campaigns/contest.js]: Pufff. Too risky.  What happen if in the futture other place has an "actions" section wiht an "a"?
Change it for: 
$("#aside-ppal > .call-to-action:first-child > .actions > a.btn")

**Review by iduetxe — CHANGES_REQUESTED** [review]

**Comment by pverdu-kuorum** [comment]
Replaced selector.

**Review by iduetxe — APPROVED** [review]

**Pull request merged**